ui/raidboss/data/04-sb/ultimate/unending_coil_ultimate.txt

```diff
@@ -75,26 +75,25 @@ hideall "--sync--"
 240.9 "Bahamut's Claw x5" duration 2.8 #Ability { id: "26B5", source: "Nael deus Darnus" }
 249.4 "Bahamut's Favor" Ability { id: "26C2", source: "Nael deus Darnus" }
 
-257.9 "Dynamo + Beam/Chariot" duration 8
+# 257.9 "Dynamo + Beam/Chariot" duration 8
 262.4 "Chain Lightning x2" Ability { id: "26C8", source: "Thunderwing" }
-#235.1 "Lunar Dynamo" Ability { id: "26BC", source: "Nael deus Darnus" }
-#238.3 "Thermionic Beam" Ability { id: "26BD", source: "Nael deus Darnus" }
+262.4 "Lunar Dynamo" Ability { id: "26BC", source: "Nael deus Darnus" }
+265.4 "Iron Chariot/Thermionic Beam" Ability { id: ["26BB", "26BD"], source: "Nael deus Darnus" }
```

Comment:
Yeah, this is a much better way to handle things. I *think* that the reason it was done this way originally had to do with limitations in the Python version of `make_timeline` and the amount of extra, manual, work that would have been needed. (As far as I know quisquous did the entirety of this one themselves and didn't have any assistance with it, so time would have been a concern.)

Reply:
Yeah, the tooling these days is way more advanced.

If I were to re-do the entire fight from scratch I'd probably use jumps for each quote to provide exact timeline timings, but frankly I hate this fight and I'll probably never touch it again after I get my clear, let alone subject the rest of my group to the amount of log gathering that would be required to get every Nael quote's timing (both in p2 and in adds phase), or the full timeline for each mini-phase of p1.